tests/sentry/api/endpoints/test_monitor_checkins.py

```diff
@@ -2,11 +2,13 @@
 
 from datetime import timedelta
 from django.utils import timezone
+from freezegun import freeze_time
 
 from sentry.models import CheckInStatus, Monitor, MonitorCheckIn, MonitorStatus, MonitorType
 from sentry.testutils import APITestCase
 
 
+@freeze_time('2019-01-01')
```

Comment:
Whoa. I didn't know about this.

Reply:
I added it before I found out we're not meant to include new dependencies 😬